poll/public/js/poll.js

```diff
@@ -18,7 +18,23 @@ function PollUtil (runtime, element, pollType) {
         }
 
         // Make gettext available in Handlebars templates
-        Handlebars.registerHelper('i18n', function(str) { return gettext(str); });
+        Handlebars.registerHelper('i18n', function(str, options) {
+            options = options.hash;
+            str = gettext(str);
+            if (options) {
+                str = str.supplant(options)
+            }
+            return str;
+        });
+
+        String.prototype.supplant = function (o) {
+            return this.replace(/{([^{}]*)}/g,
+                function (a, b) {
+                    var r = o[b];
+                    return typeof r === 'string' || typeof r === 'number' ? r : a;
```

Comment:
supplant() does variable substitution on the string. In our case it is `total`. It scans through the string looking for expressions enclosed in { } braces. If an expression is found, use it as a key on the object, and if the key has a string value or number value, it is substituted for the bracket expression